man/deepsea-stage-0.7

```diff
@@ -0,0 +1,138 @@
+.TH deepsea 7
+.SH NAME
+deepsea stage 0 \- a deeper insight
+.SH DESCRIPTION
+.B DeepSea Stage 0 (Prep)
+is a Salt orchestration and exists mainly for cluster maintenance and operational/preparation tasks. 
+.RE
+.PD
+.SH Preface
+.PP
+Stage.0 behaves differently depending on the presence of a ceph cluster. On the first invocation, or the install phase, deepsea will run all operations in parallel to speep up the process.
+Once you successfully deployed the cluster more caution is exercised and nodes are processed serially. The operation is chunked in three sections.
+.SS Master
+
+.B Validation:
+.RS
+Deepsea performs a set of system validation tests before running the ramainder of the stage.
+
+* Deepsea minion
+  See deepsea-minions (7)
+
+* master minion
+  Check for the presence of a master minion
+
+* ceph version
+  Validate if a ceph version could be found for installation
+.RE
+
+.B Salt-Api
+.RS
+In order for openattic to establish a connection with salt, hence deepsea, the salt-api needs to be enabled.
+.RE
+.B Sync
+.RS
+When a new salt or deepsea version was installed, salt needs to make sure all modules are synced over to it's minions.
+.RE
+.B Updates
+.RS
+The master minion will be patched.
+.RE
+.B Reboot
+.RS
+If Deepsea detects the need for a reboot, the master will reboot. This can be changed by adjusting
+
+.B stage_prep_master. See also deepsea-customization (7) for more information on how to change the default behavior
+.RE
+
+.RE
+.B Readycheck
+.RS
+Make sure all minions are up. Bails out after 300seconds if not all minions haven't joined back.
+.RE
+
+.SS Minions
+.B Installation of common packages
+.RS
+Deepsea installs packages are... (shouldn't this acutally go in the specfile?)
+.RE
+.B Sync
+.RS
+After a potential update was applied to the master make sure to sync the modules and grains back to the minions
```

Comment:
This sentence is vague . . . what does it mean to "apply a potential update"? How does one "sync the modules and grains back to the minions" ?